examples/federation_authority/dumps/example.json

```diff
@@ -1663,6 +1678,9 @@
         ]
     },
     "metadata": {
+      "federation_entity": {
```

Comment:
can I ask you to put also in federation_entity the following members?

organization_name
homepage_uri
policy_uri
logo_uri
contacts